.github/CODEOWNERS

```diff
@@ -8,6 +8,7 @@
 /eng/common/                    @dotnet-maestro-bot
 /eng/Versions.props             @dotnet-maestro-bot @dougbu
 /eng/Version.Details.xml        @dotnet-maestro-bot @dougbu
+PublicAPI.Unshipped.txt         @dotnet/aspnet-api-review
```

Comment:
Shouldn't this be shipped?

Reply:
Shipped should contains things that were previously reviewed and shipped. We don't need to be involved there.

Reply:
Ah I see